src/SimplePolyModel.cc

```diff
@@ -121,25 +121,25 @@ void SimplePolyModel::freezeErrorScales()
 }
 
 
-const Gtransfo& SimplePolyModel::GetTransfo(const CcdImage &Ccd) const
+const Gtransfo& SimplePolyModel::GetTransfo(const CcdImage &ccdImage) const
 {
-  // return GetMapping(Ccd)->Transfo(); // cannot do that
-  auto p = _myMap.find(&Ccd);
-  if  (p==_myMap.end()) throw LSST_EXCEPT(pex::exceptions::InvalidParameterError,"SimplePolyModel::GetTransfo, never heard of CcdImage "+Ccd.getName());
+  // return GetMapping(ccdImage)->Transfo(); // cannot do that
+  auto p = _myMap.find(&ccdImage);
+  if  (p==_myMap.end()) throw LSST_EXCEPT(pex::exceptions::InvalidParameterError,"SimplePolyModel::GetTransfo, never heard of CcdImage "+ccdImage.getName());
   return p->second->Transfo();
 }
 
-PTR(TanSipPix2RaDec) SimplePolyModel::ProduceSipWcs(const CcdImage &Ccd) const
+PTR(TanSipPix2RaDec) SimplePolyModel::ProduceSipWcs(const CcdImage &ccdImage) const
 {
-  const GtransfoPoly &pix2Tp=dynamic_cast<const GtransfoPoly&>(GetTransfo(Ccd));
-  const TanRaDec2Pix *proj=dynamic_cast<const TanRaDec2Pix*>(sky2TP(Ccd));
+  const GtransfoPoly &pix2Tp=dynamic_cast<const GtransfoPoly&>(GetTransfo(ccdImage));
```

Comment:
In this case, throwing is fine: if that's unable to be dereferenced, something has gone terribly wrong.